schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/balance/AccountUtil.java

```diff
@@ -40,26 +41,22 @@ public static Key profitAndLossAccountKey() {
         return Datastore.createKey(AccountMeta.get(), PROFIT_AND_LOSS_ACCOUNT);
     }
 
-    public static Account profitAndLossAccount() {
-        com.google.appengine.api.datastore.Transaction tx = Datastore.beginTransaction();
-        try {
-            Key id = profitAndLossAccountKey();
-            Account account = Datastore.getOrNull(AccountMeta.get(), id);
-            if (account == null) {
-                account = new Account(id);
-                account.setCurrency(DEFAULT_CURRENCY);
-                log.info("Created P&L account:{}", account);
-                Datastore.put(tx, account);
+    public static Account profitAndLossAccount() throws ModelException {
+        return TransactionOperator.execute(new ModelOperation<Account>() {
+            @Override
+            public Account execute(com.google.appengine.api.datastore.Transaction tx) {
+                Key id = profitAndLossAccountKey();
+                Account account = Datastore.getOrNull(AccountMeta.get(), id);
+                if (account == null) {
+                    account = new Account(id);
+                    account.setCurrency(DEFAULT_CURRENCY);
+                    log.info("Created P&L account:{}", account);
+                    Datastore.put(tx, account);
+                }
+                tx.commit();
+                return account;
```

Comment:
I think throwing ModelException from this method is a mistake.  It gets propagated to many places and makes "caller"  blind in terms of what the problem is.  The real problem is ModelException blinds callers from what types of exceptions methods can throw.  I would suggest either of

1. Create a new `ModelOperationNoEx` that doesn't throw
```java
interface ModelOperationNoEx<T> extends TransactionOperation<T, RuntimeException> {
}
```
2. Use the convenience method `TransactionOperator.executeNoEx`

But I'm not OK with changing signatures of so many methods to throw ModelException.  Please fix and RePush.